StudioCore/Resource/ResourceLoadPipeline.cs

```diff
@@ -66,13 +67,16 @@ public ResourceLoadPipeline(ITargetBlock<ResourceLoadedReply> target)
             try
             {
                 var res = new T();
-                bool success = res._Load(r.File, r.AccessLevel, r.GameType);
+                bool success = false;
```

Comment:
I don't think this needs to be initialized to false. The value is only relevant if there isn't an exception.